integration-tests/src/test_loop/builder.rs

```diff
@@ -112,6 +112,11 @@ impl TestLoopBuilder {
         self
     }
 
+    pub fn stores_override_hot_only(mut self, stores: Vec<Store>) -> Self {
```

Comment:
What is it for?

Reply:
It's just so that it's less awkward to use for tests that don't override the cold storage.